src/v/ssx/tests/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ rp_test(
     sleep_abortable_test.cc
     task_local_ptr_test.cc
     watchdog_test.cc
+    when_all_test.cc
```

Comment:
should this be added to the bazel build files as well?
Whichever test file i was copying from doesn't seem to be present in the BUILD file.

Reply:
Yes we should port the test to Bazel. I just think we haven't had time to port these tests. Do you think you could give it a shot?

Reply:
I _think_ the general push is to "bazelize" all tests, so I'd recommend creating a new test target as in https://github.com/redpanda-data/redpanda/blob/ef579f2d4af960f6cf489fe4b6f13256f953ffbe/src/v/ssx/tests/BUILD#L49-L63

Reply:
sorry, tyler. my UI was out of sync 😇